login/login.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <noscript>To play CardShifter, please either enable JavaScript or use a JavaScript-enabled browser.</noscript>
+        <form name = "login_information">
+            Server:
+            <input name = "server" type = "text" placeholder = "dwarftowers.com" />
```

Comment:
I think placeholder would be better as `127.0.0.1` for local server. We can add other servers into the body of the document, if needed.